modules/NEON/README.md

```diff
@@ -1,21 +1,23 @@
 # RAD Lab NEON Module
 
+## Module Overview
+
 This RAD Lab module provides a managed environment for research and data analytics on publicly shared data of the National Ecological Observatory Network (NEON) by subscribing to NEON Big Query dataset via Analytics Hub and exploring it via Jupyter Notebook(s) running over Vertex AI workbench.
 
 ## GCP Products/Services 
 
-* Vertex AI Workbench Notebooks (a.k.a. AI Platform Notebooks)
-* Virtual Private Cloud (VPC)
-* BigQuery
-* Analytics Hub
-* Cloud Storage
-* Billing Budget
+1. Vertex AI Workbench Notebooks
+2. Virtual Private Cloud (VPC)
+3. BigQuery
+4. Analytics Hub
+5. Cloud Storage
+6. Billing Budget
```

Comment:
I think the *unordered* (original) style makes more sense here.

Reply:
Done